Makefile

```diff
@@ -20,18 +22,17 @@ install-dev-dependencies:
 	bin/composer install
 
 integration-tests:
-	docker-compose exec app bats -r tests/
+	docker-compose exec -T app bats -r tests/
 
 unit-tests: install-dev-dependencies
-	docker-compose exec app composer test
+	docker-compose exec -T app composer test
 
 up:
 	docker-compose up -d
-	@for i in `seq 1 10`; do docker-compose exec app true && exit 0; echo "Waiting for docker ready..."; sleep 2; done; exit 1
-	@docker-compose exec app docker/wait_for_db
-	mkdir -p ./uploads && docker-compose exec app chmod 777 ./uploads
-	docker-compose exec app chmod 777 ./archive
-
+	@for i in `seq 1 10`; do docker-compose exec -T app true && exit 0; echo "Waiting for docker ready..."; sleep 2; done; exit 1
+	@docker-compose exec -T app docker/wait_for_db
```

Comment:
We should probably find a way for the app to check and set its own permissions on startup rather than having the Makefile be responsible for that. Not enough to hold up this PR, but something to think about for future.